websites/M/MoodlePresence/dist/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+        "name": " .•` KIΛRΛ `•. ",
+        "id": "586905336850677760"
+    },
+    "contributors": [
+        {
+            "name": "ΛΚΘRΛ",
+            "id": "387655649934311427"
+        }
+    ],
+    "service": "Moodle",
+    "version": "1.0.0",
+    "color": "#ffbc4f",
+    "description": {
+        "en": "Your end-to-end online learning platform for K12, higher education and workplace training. The most used open source LMS in the world."
+    },
+    "tags": [
+        "moodle",
+        "education",
+        "media"
+    ],
+    "category": "socials",
+    "url": "moodle.com",
+    "logo": "https://imgur.com/a/q2jcE68",
```

Comment:
Use the form "https://i.imgur.com/ABCDEF.png" aka remove "/a", add "i." before imgur.com, and add ".png" at the end

Reply:
Use https://i.imgur.com/ltzuG6b.png